doc/man1/openssl-fipsinstall.pod.in

```diff
@@ -190,6 +191,11 @@ See RFC 7627 for information related to EMS.
 Configure the module to not allow truncated digests to be used with Hash and
 HMAC DRBGs.  See FIPS 140-3 IG D.R for details.
 
+=item B<-tdes_encrypt_disabled>
+
+Configure the module to not allow Triple-DES encryption. (Triple-DES decryption
```

Comment:
Ditch the parenthesis.  A full sentence should never be parenthesised by itself.